classification/migrations/0107_alter_discordancenotification_options_and_more.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 4.2.2 on 2023-08-22 02:14
+
```

Comment:
Something worth doing if you've made 2 migrations (where the 2nd is just an extension of the first).
Run

python3 manage.py migrate classification 0105
(this will revert your database to how it was before making discordance notifications)

then delete 0106_discordancen... and 0107_alter_discordance...

now run
python3 manage.py migrate classification

You'll end up with one migration script that changes the schema only once.